include/zephyr/drivers/sensor.h

```diff
@@ -168,6 +168,8 @@ enum sensor_channel {
 	SENSOR_CHAN_GAUGE_TIME_TO_EMPTY,
 	/** Time to full in minutes **/
 	SENSOR_CHAN_GAUGE_TIME_TO_FULL,
+	/** If the battery is being charged or discharged */
+	SENSOR_CHAN_GAUGE_CHARGING_STATE,
```

Comment:
I don't think we want or need this. We can just check the charging current property.